CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+# 0.4.0
+## Major changes and new features
+- Add filtering of pipelines on Kedro-Viz flowchart in VSCode.
+- Add documentation for filter pipeline on Kedro-Viz flowchart.
```

Comment:
Can we also add Expand/Collapse of pipelines on kedro-viz flowchart on vscode here, because it is not there in the previous release although we pushed it; there was some issue getting it to work so we didn't announce it to users